docs/advanced/operators.md

```diff
@@ -42,24 +42,10 @@ export class AnimalsState {
 ```
 
 The `patch` operator expresses the intended modification quite nicely and returns a function that will apply these modifications as a new object based on the provided state.
-In order to understand what this is doing let's express this in a long handed form:
```

Comment:
Could we add this back but using the spread operator in the code?

Reply:
Sure

Reply:
That's exactly what I meant in my previous comment. 👍